burr/core/persistence.py

```diff
@@ -88,7 +88,11 @@ class BaseStatePersister(BaseStateLoader, BaseStateSaver, metaclass=ABCMeta):
     Extend this class if you want an easy way to implement custom state storage.
     """
 
-    pass
+    def is_initialized(self) -> bool:
+        """Check if the persister has been initialized. Default behavior is False.
+        Persisters that require initialization can override this method.
+        """
+        return False
```

Comment:
this should raise a `NotImplementedError`.

Then we need the subclasses of this that are in Burr to implement this -- and much like the SQLLite set some internal value.